MODULE.bazel

```diff
@@ -46,3 +49,19 @@ use_repo(
     "amazon_corretto_java_image_base",
     "ubuntu_mantic",
 )
+
+# https://github.com/bazelbuild/rules_python/pull/713#issuecomment-1885628496
+# Satisfy running tests in Docker as root.
+bazel_dep(name = "rules_python", version = "0.24.0")
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+python.toolchain(
+    configure_coverage_tool = False,
+    ignore_root_user_error = True,
+    python_version = "3.11",
+)
```

Comment:
Not sure I see the connection between adding python here and rules_pkg

Reply:
`rules_pkg` 0.10.1 depends on `rules_python`
https://registry.bazel.build/modules/rules_pkg , we're configuring the `ignore_root_user_error` for CI, or, we can invest in running CI as non-root user

Reply:
Got it, I recall this discussion now.